### PR TITLE
XAMARIN-43417-Need to prepare UG content for fontname Usability Feature

### PR DESCRIPTION
**Pull request by Benin-JS**
Hi Team,

I have added the content for font name usability feature and Style Sheet Feature with code snippets . Kindly review the changes and let me know your feedback.

With Regards,
Benin JS

### CODE REVIEW

File: Xamarin/Rich-Text-Editor/Basic-Features.md [hunk 1]
```diff
@@ -151,7 +166,7 @@ private void Richtexteditor_HyperlinkSelected(object sender, HyperlinkSelectedEv
 
 ## Cursor Position
 
- Rich Text Editor provides the support to get or set the current cursor position based on the character count in the rich text editor. The following code example explains how to get or set a cursor position in the Rich Text Editor.
+ Rich Text Editor provides the support to get or set the current cursor position based on the character count in the rich text editor. The following code example explains how to get or set a cursor position in the Rich Text Editor .
```
Comment (Mohanaraja) [inline Xamarin/Rich-Text-Editor/Basic-Features.md:169]: Remove the unwanted code changes

File: Xamarin/Rich-Text-Editor/Basic-Features.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ The Xamarin Rich Text Editor control displays the text/formatted text(HTML strin
 {% highlight C# %} 
 richtexteditor = new SfRichTextEditor()
 {
-	VerticalOptions = LayoutOptions.FillAndExpand,			
+	VerticalOptions = LayoutOptions.FillAndExpand ,
```
Comment (Mohanaraja) [inline Xamarin/Rich-Text-Editor/Basic-Features.md:27]: Remove the unwanted code changes

File: Xamarin/Rich-Text-Editor/Basic-Features.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ documentation: ug
 
 ## Setting Text
 
-The Xamarin Rich Text Editor control displays the text/formatted text(HTML string) that can be set using the [`Text`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.RichTextEditor.SfRichTextEditor.html#Syncfusion_XForms_RichTextEditor_SfRichTextEditor_Text) property.
+The Xamarin Rich Text Editor control displays the text /formatted text(HTML string) that can be set using the [`Text`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.RichTextEditor.SfRichTextEditor.html#Syncfusion_XForms_RichTextEditor_SfRichTextEditor_Text) property.
```
Comment (Mohanaraja) [inline Xamarin/Rich-Text-Editor/Basic-Features.md:14]: Remove the unwanted code changes

**Review by Mohanaraja — CHANGES_REQUESTED** [review]
Address the feedback

File: Xamarin/Rich-Text-Editor/Basic-Features.md (hunk 1 above)
Comment (Benin-JS) [reply Xamarin/Rich-Text-Editor/Basic-Features.md]: Removed the unwanted changes.

File: Xamarin/Rich-Text-Editor/Basic-Features.md (hunk 2 above)
Comment (Benin-JS) [reply Xamarin/Rich-Text-Editor/Basic-Features.md]: Removed

File: Xamarin/Rich-Text-Editor/Basic-Features.md (hunk 3 above)
Comment (Benin-JS) [reply Xamarin/Rich-Text-Editor/Basic-Features.md]: Removed.

**Review by Mohanaraja — APPROVED** [review]
changes are fine

**Pull request merged**